README.md

```diff
@@ -215,6 +132,8 @@ We use the [Development Project Board](https://github.com/orgs/getAlby/projects/
 - Check out the issues that have specifically been [marked as being friendly to new contributors](https://github.com/getAlby/lightning-browser-extension/issues?q=is%3Aopen+is%3Aissue+label%3Adesign+label%3A%22good+first+issue%22)
 - You can also review open PRs
 
+- [Contribution guide for new developers](./doc/CONTRIBUTION.md)
```

Comment:
```suggestion
- Check out the issues that have specifically been [marked as being friendly to new contributors](https://github.com/getAlby/lightning-browser-extension/issues?q=is%3Aopen+is%3Aissue+label%3Adesign+label%3A%22good+first+issue%22)
- You can also review open PRs
- [Contribution guide for new developers](./doc/CONTRIBUTION.md)
```

Reply:
Remove the extra line to avoid empty line between list-items:
![image](https://user-images.githubusercontent.com/1016218/205250303-71da8e42-24d4-4f16-95ba-2349abbaeb14.png)